inscriptions/ISic030013.xml

```diff
@@ -177,38 +180,77 @@
       <desc>Add Description Here</desc>
     </graphic>
   </surface>
-
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#TS"> <!-- add text, apparatus, translation and commentary -->
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#SN"> <!-- add text, apparatus, translation and commentary -->
                 <ab>
-                    <lb n="1"/>
-                    <lb n="2"/>
+                    <lb n="1"/> ἐ<unclear>π</unclear>ὶ ἐξο̄<unclear>λ</unclear>είαι καὶ αὐτο͂ν καὶ γενεᾶς· Νίκυλλος
```

Comment:
add around all the names: <persName><name>Νίκυλλος</name></persName> (change/check previous files)